### DISCUSSION TITLE
Welcome to convert2rhel Discussions!

### DISCUSSION THREAD

**Discussion by bocekm** [oamg/convert2rhel#192] [General]
We’re using Discussions as a place to connect with other members of the community and users of convert2rhel in general. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with others.

Maybe you just want to contribute and you are unsure where to start. Talk to us here and we'll guide you.

**Comment by Jc114013** [comment]
Convert2Rhel CentOS8 to RHEL 8 in the middle of the conversion I lose network on the VM... It can't even ping itself at that point

**Reply by bocekm** [reply]
Hi @Jc114013, I haven't encountered this issue before. You can [open a support case with Red Hat Customer Support](https://access.redhat.com/support/cases/) so they can help you find the root cause.

**Comment by Jc114013** [comment]
This was due to the fact that firewalld was running on this VM. Thank You
